### PR TITLE
Move some redundant code to a helper function

### PR DESCRIPTION
**Pull request by bcrochet**
Signed-off-by: Brad P. Crochet <brad@redhat.com>

### CODE REVIEW

**Review by samira-barouti — APPROVED** [review]

File: certification/engine/engine.go [hunk 1]
```diff
@@ -102,35 +102,26 @@ var operatorPolicyChecks = map[string]certification.Check{
...
+
+	return checks
 }
 
-func OperatorPolicy() []string {
-	checks := make([]string, len(operatorPolicyChecks))
-	i := 0
+func AllChecks() []string {
+	return makeCheckList(nameToChecksMap)
+}
```
Comment (komish) [inline certification/engine/engine.go:119]: This is a test comment

File: certification/engine/engine.go [hunk 2]
```diff
@@ -102,35 +102,26 @@ var operatorPolicyChecks = map[string]certification.Check{
...
-func AllChecks() []string {
-	all := make([]string, len(nameToChecksMap))
+func makeCheckList(checkMap map[string]certification.Check) []string {
+	checks := make([]string, len(checkMap))
 	i := 0
 
-	for k := range nameToChecksMap {
-		all[i] = k
+	for key := range operatorPolicyChecks {
+		checks[i] = key
```
Comment (komish) [inline certification/engine/engine.go:110]: This is a review

**Review by komish — COMMENTED** [review]
redacted (test comment was here)

**Review by komish — APPROVED** [review]
/lgtm

**Pull request merged**